app/models/log.rb

```diff
@@ -7,13 +7,14 @@ def set_uid
 
   def write(trace = nil, &block)
     text = trace.nil? ? yield : trace
-    Log.create!(attributes.merge(text: text))
+    Log.create!(attributes.except('id').merge(text: text))
```

Comment:
all changes in this model are just porting the Pro version to CE
The Pro version fixes this case:
```
l = Log.new.write('hello')
l.write 'hello 2' # raises unique id error
```

Reply:
Can you explain this more since `Log.new.write('hello')` seems to work for both CE and Pro on my end. Also, how is this related to the PR?

Reply:
If you apply the lines above together (`write` on the result of a `write`), you should see an error in CE.
Here we are just porting the Log model code from Pro to CE, since that problem was fixed in Pro before but forgot to port it to CE.
We need it fixed in this PR because we are passing to the bg job as a logger the result of a `write`:  https://github.com/dradis/dradis-ce/pull/489/files#diff-5d55ddbe9a8496e5b77d56dbb8004988R34 (to keep consistency with Pro)